SCClassLibrary/Common/Control/ServerStatus.sc

```diff
@@ -142,7 +92,8 @@ ServerStatusWatcher {
 			} {
 				0.2.wait;
 			};
-			if(serverRunning.not, {
+
+			if(server.serverRunning.not, {
```

Comment:
just for consistency, we should keep one format over the whole file:

```
if(server.serverRunning.not) {
```

Reply:
thanks for spotting - redirected doWhenBooted to server:doWhenBooted, so that line is gone.